### ISSUE TITLE
Playback speed support

### ISSUE THREAD

**Issue by rush-skills** [zohararad/audio5js#75]
Does this plugin support controlling of playback speed?
If not are there any plans to do so?

**Comment by zohararad** [comment]
no, it doesn't support this at the moment. I'm happy to accept a PR if you want to provide one.

**Comment by rush-skills** [comment]
I was able to implement it in html5 audio using their native playbackRate setting.
I did this in my code to make it work:
> audio5js.audio.audio.playbackRate = newRate;
Not sure how to do this incase of flash.

**Comment by zohararad** [comment]
Me neither actually... not very experienced with AS3. Anyhow, I'm happy to accept this for HTML5 only as a PR if you want to send one.
Otherwise, and perhaps more elegant - just add this after initializing?
